tests/AttendanceHandlerTests.cpp

```diff
@@ -0,0 +1,37 @@
+#include <gtest/gtest.h>
+
+#include "Attendance.hpp"
+#include "AttendanceHandler.hpp"
+#include "Calender.hpp"
+
+struct AttendanceHandlerTests : public ::testing::Test
+{
+    AttendanceHandler attendanceHandler_;
+};
+
+TEST_F(AttendanceHandlerTests, Can_Report_Absence_And_View_report)
+{
+    Lesson lesson("2019-07-06", "16:30", "temp Subject");
+    User user("Kamil", "Kami", User::Group::evening, "Kami", "Kami", "k.w2@gmail.com", "xx");
```

Comment:
lesson and user can be moved to the fixture AttendanceHandlerTests

Reply:
moved